### PR TITLE
Adding among us ejecting feature

### PR DESCRIPTION
**Pull request by slocknad**
A mod-only feature that works just like kicking but it's based off the popular game Among Us

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -56,29 +56,6 @@ async def help(ctx, *arg):
     if ctx.author.dm_channel != ctx.channel:
         await ctx.send(ctx.author.mention + ", I've sent you a DM with the help menu.")
 
-
-def checkdb(t):
```
Comment (martin-sweeny) [inline bot.py:60]: What's going on here? Why'd this get removed?

**Comment by martin-sweeny** [comment]
image

**Review by martin-sweeny — COMMENTED** [review]
If we're being anal, the `checkdb` removal _should_ be in its own PR. I won't reject the PR for that, but if you want to split it into two PRs that'd be the "better" way

File: bot.py (hunk 1 above)
Comment (slocknad) [reply bot.py:60]: OH 100% UNINTENTIONAL
I was working on a separate PR and I guess I carried the changes

Comment (martin-sweeny) [reply bot.py:60]: Shit happens lol

**Review by martin-sweeny — APPROVED** [review]
fuck it lets do it live

**Pull request merged**